src/Infrastructure/Controller/ErrorController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
```

Comment:
I believe you should add the copyright notice to this file.

Reply:
Is still WIP, this will be fixed in: https://github.com/OpenConext/Stepup-ReadID/pull/27, after the legal team is consulted.

Reply:
:+1: